src/solver/main.cpp

```diff
@@ -104,8 +104,9 @@ bool SolverApplication::prepare(int argc, char* argv[])
     options.usedByTheSolver = true;
 
     // Parse arguments and store usefull values
-    if (not GrabOptionsFromCommandLine(argc, argv, pSettings, options))
-        return false;
+    int ret = GrabOptionsFromCommandLine(argc, argv, pSettings, options);
```

Comment:
variable **ret** not very useful here, as it is not used elsewhere in this function.
Maybe the old code is more concise and clear :
```c++
if (not GrabOptionsFromCommandLine(argc, argv, pSettings, options))
        return false;
```

Reply:
done